### DISCUSSION TITLE
limit by temp.

### DISCUSSION THREAD

**Discussion by budianto7** [zqhong/RyzenTuner#23] [Q&A]
is there a way to limit temp just like in ryzenadj?

**Comment by zqhong** [comment]
That’s a good suggestion. The main issue right now is that it would require some UI changes, but I’ll consider adding a temp limit feature like RyzenAdj when I have time. Thanks!

**Comment by zqhong** [comment]
The new version already supports this feature, and I will release it after testing.

Please note that this is a newly released version and may contain bugs. Feedback is welcome.
